Update key-vault-group-permissions-for-apps.md

ld0614 opened the pull request:
The PowerShell v5 module uses PermissionsToSecrets rather than PermissionToSecrets

azurecla commented:
Hi __@neumanndaniel__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

deneha commented:
@neumanndaniel Thank you for this contribution.  I'll have the author review.

@KundanaP  Would you please review this contribution and let me know if I should merge it?  Thank you.

neumanndaniel commented:
@deneha @KundanaP It is a bit disappointing to see that nothing has happened the last 18 days on that topic.

KundanaP commented:
@neumanndaniel @deneha  -  I'm very sorry that I missed this notification. The article looks great! Thanks for writing. There seems to be a minor typo in this sentence " I assume that you have a ready to use IaaS VM ARM template". Rest looks great.

deneha commented:
@neumanndaniel If you can make that one change @KundanaP suggested and fix the merge conflict on this request, I'll get this merged for you right away.

neumanndaniel commented:
@deneha @KundanaP My article was only for explaining why I have done the code change for the setting workspaceKey. It is enough for me when you merge my code change.

neumanndaniel commented:
Ok. I just checked the documentation website. The code change has been done already.
![capture1](https://cloud.githubusercontent.com/assets/13654904/9959053/7a98001a-5e10-11e5-9083-72306138d744.PNG)

neumanndaniel commented:
And you are using the wrong settings:

It is "workspaceId" instead of "workspace_name".

It is "workspaceKey" instead of "workspace_key".

deneha commented:
@KundanaP Additional feedback here - please check

richrundmsft commented:
Thanks for pointing out that the field names are incorrect. You are totally right - it should be workspaceId and workspaceKey.

BTW, thanks for writing your blog post - I know it helped to resolve at least one customer issue.

ld0614 committed:
Update key-vault-group-permissions-for-apps.md

The PowerShell v5 module uses PermissionsToSecrets rather than PermissionToSecrets
